common/plugins/eu.esdihumboldt.hale.common.align.merge/META-INF/MANIFEST.MF

```diff
@@ -23,7 +23,7 @@ Import-Package: com.google.common.base;version="17.0.0",
  eu.esdihumboldt.hale.common.instance.model,
  eu.esdihumboldt.util,
  eu.esdihumboldt.util.definition,
- javax.annotation,
+ javax.annotation;version="[1.2.0,1.2.0]",
```

Comment:
Are there now multiple versions flying around? Might the hard cap on the version cause problems later?

Reply:
This is a remnant from my attempts to move to JDK 11 where I pulled in `javax:javaee-api:7.0.0` which also defined this package, but only partially. From the commit message: "This prevents inadvertently importing the package from recursive dependencies where the package cannot be easily excluded (e.g. `javax:javaee-api:7.0.0`)."

If the version of the `javax.annotation` package is updated in the future, the version pin needs to be updated, too. I didn't think that to be a problem as it should result in a compile error if 1.2.0 isn't available anymore.

Given that we abandoned the move to JDK 11 for now, I wouldn't mind reverting it now and revisiting it later.

Reply:
No need to revert in my view, was just interested in the reason.